frontend/src/lib/utils.tsx

```diff
@@ -723,20 +723,22 @@ export function autoCaptureEventToDescription(
     }
 
     const getVerb = (): string => {
-        if (event.properties.$event_type === 'click') {
-            return 'clicked'
+        const eventTypesMap: { [key: string]: string } = {
```

Comment:
Make this a private and global Dict instead? Otherwise, we would create a new Dict on every method call